gcalcli/ics.py

```diff
@@ -64,65 +64,54 @@ def CreateEventFromVOBJ(
 
         event['recurrence'] = ['RRULE:' + ve.rrule.value]
 
-    if hasattr(ve, 'dtstart') and ve.dtstart.value:
```

Comment:
Combined this check w/ the one above and dedented this huge if block.

Technically if an event had `.dtstart` but not `.dtstart.value` this change might skip that event more explicitly, but I'm pretty sure that scenario was never valid and nobody was relying on that corner case behavior.